<?xml version="1.0" ?>
<launch>
	<!-- declare arg to be passed in -->
	<arg name="map"/>
	<arg name="machine" default="localhost" />
	<arg name="user" default="" />
        <arg name="launch_manager" default="true"/>

	<machine name="$(arg machine)" address="$(arg machine)" env-loader="$(optenv ROS_ENV_LOADER )" user="$(arg user)" default="true"/>

 	<!--node pkg="topological_navigation" type="map_publisher.py" name="topological_map_publisher" args="$(arg map)"/-->
	<node pkg="topological_navigation" type="map_manager.py" name="topological_map_manager" args="$(arg map)" respawn="true" if="$(arg launch_manager)"/>

	<node pkg="topological_navigation" type="visualise_map.py" name="visualise_map" args="$(arg map)" respawn="true"/>

</launch>
